gvr-immersivepedia/app/src/main/java/org/gearvrf/immersivepedia/Main.java

```diff
@@ -81,19 +82,10 @@ public void run() {
             }
         });
         GVRControllerType[] controllerTypes = new GVRControllerType[] { GVRControllerType.GAZE, GVRControllerType.CONTROLLER  };
-        gvrContext.getInputManager().selectController(gvrContext, controllerTypes,  new GVRInputManager.ICursorControllerSelectListener()
+        gvrContext.getInputManager().selectController(controllerTypes,  new GVRInputManager.ICursorControllerSelectListener()
```

Comment:
selectController makes useGazeCursorController somewhat redundant. I guess we either remove useGazeCursorController from gvr.xml or put more things there; like ``"cursorController="gaze|controller"``..